lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -126,6 +126,55 @@ class EngineAccessibilityFeatures implements ui.AccessibilityFeatures {
     }
     return EngineAccessibilityFeatures(value);
   }
+
+  EngineAccessibilityFeaturesBuilder getBuilder(){
+    return EngineAccessibilityFeaturesBuilder(_index);
+  }
+}
+
+class EngineAccessibilityFeaturesBuilder {
```

Comment:
Let's write some tests for this class to make sure that it's setting the correct flags.